x-pack/plugins/cases/public/components/case_view/components/case_view_alerts.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { Case } from '../../../../common';
+import { useKibana } from '../../../common/lib/kibana';
+import { getManualAlertIds, getFeatureIdsFromAlertIndices } from '../../user_actions/helpers';
+
+interface CaseViewAlertsProps {
+  caseData: Case;
+}
+export const CaseViewAlerts = ({ caseData }: CaseViewAlertsProps) => {
+  const { triggersActionsUi } = useKibana().services;
+
+  const alertIdsQuery = useMemo(
+    () => ({
+      ids: {
+        values: getManualAlertIds(caseData.comments),
+      },
+    }),
+    [caseData.comments]
+  );
+  const alertFeatureId = useMemo(
+    () => getFeatureIdsFromAlertIndices(caseData.comments),
+    [caseData.comments]
+  );
+
+  const alertStateProps = useMemo(() => {
+    const configurationId = alertFeatureId.includes('siem') ? 'securitySolution' : 'observability';
+    return {
+      alertsTableConfigurationRegistry: triggersActionsUi.alertsTableConfigurationRegistry,
```

Comment:
@XavierM can't the triggersUI component consume itself? I find it redundant to send this registry as a prop when the triggersActionsUi can import it directly from the kibana services object.

Reply:
This one is annoying because the useKibana inside of trigger actions is not aware of the alertsTableConfigurationRegistry when consumed by another plugin. I will think more about it to find a better solution if you have one I am here to hear it

Reply:
If I understand it correctly. the register is an object that is put inside the StartUIPlugins and SetupUIPlugins which are singletons.

Shouldn't you be able to access those from inside trigger ui?